README.md

```diff
@@ -105,8 +112,15 @@ Results are collected just by running the specs.
 
 #### SQLite3
 
-By default, profiles are collected in an SQL database. Make sure you've
-run the installation rake task before attempting.
+Make sure you've run the installation rake task before attempting.
+
+You can configure `RspecProfiling` to collect results in a SQL database in `config/initializers/rspec_profiling.rb`:
```

Comment:
For later: is there ever a world where we don't want `rails` as a dependency?

Reply:
It's a good possibility I think we can capture that as a future issue.